share/agrammon-rest.openapi

```diff
@@ -0,0 +1,150 @@
+    /model/latex:
+        get:
+            summary: Get LaTeX model documentation
+            operationId: getLatex
+            parameters:
+                - in: query
+                  name: technical
+                  schema:
+                      type: string
+                - in: query
+                  name: sort
+                  schema:
+                      type: string
+                      enum:
+                          - 'model'
+                          - 'calculation'
+            responses:
+                '200':
+                    description: LaTeX file
+                    content:
+                        text/plain:
+                            schema:
+                                type: string
+                '401':
+                    description: Not authorized.
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/Unauthorized"
+    /run:
+        post:
+            summary: Run simulation
+            operationId: runSimulation
+            requestBody:
+                required: true
+                content:
+                    multipart/form-data:
+                        schema:
+                            $ref: "#/components/schemas/Run"
+                        encoding:
+                            inputs:
+                                contentType: text/csv
+            responses:
+                '200':
+                    description: Simulation results
+                    content:
+                        application/json:
+                            schema:
+                                description: JSON formatted
+                        text/csv:
+                            schema:
+                                description: CSV formatted
+                        text/plain:
+                            schema:
+                                description: text formatted
+                '401':
+                    description: Not authorized.
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/Unauthorized"
+components:
+    schemas:
+        Run:
+            type: object
+            required:
+                - simulation
+                - dataset
+                - inputs
+            properties:
+                simulation:
+                    type: string
+                dataset:
+                    type: string
+                inputs:
+                    type: string
+                technical:
+                    type: string
+                model:
+                    type: string
+                variants:
+                    type: string
+                print-only:
+                    type: string
+                language:
+                    type: string
+                format:
+                    type: string
```

Comment:
format went away in favor of the accept header, so should be removed here also

Reply:
Done.